addon/adapters/user.js

```diff
@@ -1,4 +1,20 @@
 import OsfAdapter from './osf-adapter';
+import Ember from 'ember';
 
 export default OsfAdapter.extend({
+    findHasMany(store, snapshot, url, relationship) {
```

Comment:
Apologies for the noise. Some further experimentation suggested that we may be able to use the following after all:
`return this.store.query('account', {q:[1,2,3,4]});`

to get a url of the form needed, eg

`https://staging-metadata.osf.io/v1/id/collections/experimenter.accounts/_search?q=1&q=2&q=3&q=4`

This would let Dawn do exactly what she needs in a simpler fashion. (sorry for the noise- ember specs, docs, and various methods aren't always consistent with handling of arrays) She'll try it out as a direct path and report back.

If it doesn't work out, the path forward is to merge this now, but figure out API in the future before embeds come up again.